coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -152,9 +172,25 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
             raise KeyError("Language {!r} is not defined for docstyle {!r}."
                            .format(language, docstyle))
 
+        def get_from_docstyle_settings(*args):
+            values = list()
+            for req_setting in args:
+                setting = str(
+                    docstyle_settings.get(req_setting, ""))
+                values.append(setting)
+            return values
```

Comment:
can be shortened :)
```python
return [str(docstyle_settings.get(req_setting, "")) for req_setting in args]
```
btw dou you need to invoke `str()` on `setting.get()`? isn't already a string returned?